src/test/java/com/documents/WithdrawalDocumentRequestServiceTest.java

```diff
@@ -0,0 +1,196 @@
+        assertEquals(Long.valueOf(1234), savedWithdrawalDocumentRequest.getNumberSheet());
+    }
+
+    @Test
+    public void behavioural_withdrawalDocumentRequest_findById_should_return_true() throws Exception {
+
+        WithdrawalDocumentRequest withdrawalDocumentRequestToSave = new WithdrawalDocumentRequest();
+
+        when(withdrawalDocumentRequestRepository.findOne(any(long.class))).thenReturn(withdrawalDocumentRequestToSave);
+
+        withdrawalDocumentRequestToSave = withdrawalDocumentRequestServiceImpl.findById((long) 123);
+
+        Assert.assertNotNull(withdrawalDocumentRequestToSave);
+    }
+
+    @Test
+    public void behavioural_withdrawalDocumentRequest_deleteById_should_return_true() throws Exception {
+
+        WithdrawalDocumentRequest withdrawalDocumentRequest = new WithdrawalDocumentRequest();
+
+        withdrawalDocumentRequest.setId((long) 3);
+
+        withdrawalDocumentRequestServiceImpl.delete(withdrawalDocumentRequest.getId());
+
+        WithdrawalDocumentRequest findedWithdrawalDocumentRegistrationForm = withdrawalDocumentRequestServiceImpl.findById((long) 3);
+
+        assertNull(findedWithdrawalDocumentRegistrationForm);
+
+    }
+
+    @Test
+    public void functionality_withdrawalDocumentRequest_deleteById_should_return_delete_withdrawalDocument() throws Exception {
+
+        //Act
+        WithdrawalDocumentRequest withdrawalDocumentRequest = new WithdrawalDocumentRequest();
+
+        withdrawalDocumentRequestServiceImpl.delete(withdrawalDocumentRequest.getId());
+
+        //Assert
+        verify(withdrawalDocumentRequestRepository).delete(withdrawalDocumentRequest.getId());
+
+    }
+
+    @Test
+    public void behavioural_withdrawalDocumentRequest_findAll_should_return_true() throws Exception {
+
+        List<WithdrawalDocumentRequest> withdrawalDocumentRequests = new ArrayList<>();
+
+        WithdrawalDocumentRequest withdrawalDocumentRequest = new WithdrawalDocumentRequest();
+
+        setup(withdrawalDocumentRequest);
+
+        when(withdrawalDocumentRequestRepository.findAll()).thenReturn(withdrawalDocumentRequests);
+
+        withdrawalDocumentRequests.add(withdrawalDocumentRequest);
+
+        List<WithdrawalDocumentRequest> findedForms;
+        findedForms = withdrawalDocumentRequestServiceImpl.findAll();
+
+        Assert.assertNotNull(findedForms);
+
+    }
+
+    @Test
+    public void functionality_withdrawalDocumentRequest_findAll_should_return_list_of_withdrawalDocumentRequest() throws Exception {
+
+        List<WithdrawalDocumentRequest> withdrawalDocumentRequests = new ArrayList<>();
+
+        WithdrawalDocumentRequest withdrawalDocumentRequest = new WithdrawalDocumentRequest();
+
+        setup(withdrawalDocumentRequest);
+
+        when(withdrawalDocumentRequestRepository.findAll()).thenReturn(withdrawalDocumentRequests);
+
+        withdrawalDocumentRequests.add(withdrawalDocumentRequest);
+
+        List<WithdrawalDocumentRequest> findedForms;
```

Comment:
rename this variable from findedForms into foundForms. Sounds more like english like this :D